pkg/networkserver/utils.go

```diff
@@ -151,11 +154,60 @@ func nextClassADataDownlinkSlot(dev *ttnpb.EndDevice) (time.Time, bool) {
 	}
 }
 
-func nextConfirmedClassCDownlinkAt(dev *ttnpb.EndDevice, defaults ttnpb.MACSettings) time.Time {
-	if dev.GetMACState().GetLastConfirmedDownlinkAt() == nil {
-		return time.Time{}
+const (
```

Comment:
@johanstokking please confirm that I got the spec right here.
Also the const durations - e.g. the `tBeaconDelay`:
1.0.3 spec:
![2019-11-19-17:03:08-screenshot](https://user-images.githubusercontent.com/12877905/69163445-7f275600-0aee-11ea-9c57-dd206409e52f.png)

`mSec` stands for `microsecond` here, right?